generators/client/templates/vue/webpack/webpack.common.js.ejs

```diff
@@ -7,7 +7,6 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const MergeJsonWebpackPlugin = require('merge-jsons-webpack-plugin');
 <%_ } _%>
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const { BaseHrefWebpackPlugin } = require('base-href-webpack-plugin');
```

Comment:
Couldn't find any errors when removing it as the href plugin is not compatible with html-webpack-plugin >= 4.x